src/test/scala/com/gu/zuora/ZuoraSpec.scala

```diff
@@ -46,25 +49,31 @@ class ZuoraSpec extends AsyncFlatSpec with Matchers {
     }
   }
 
-  it should "be able to find a monthly recurring subscription" in {
-    uatService.getRecurringSubscription("30001758", Monthly).map {
-      response =>
-        response.isDefined should be(true)
-        response.get.ratePlans.head.productName should be("Contributor")
+  it should "be able to find a monthly recurring subscription just after creation" in {
+    val justAfterCreation = LocalDateTime.of(2017, 12, 7, 15, 47, 22)
+    GetRecurringSubscription(uatService, justAfterCreation, "30001758", Monthly).map {
+      _.flatMap(_.ratePlans.headOption.map(_.productName)) should be(Some("Contributor"))
```

Comment:
this was just because I was getting "false should be true" type messages from the tests which was a bit unsatisfying